CharlieBackend.Api.UnitTes/LessonServiceTests.cs

```diff
@@ -178,142 +278,58 @@ public async Task CreateLessonAsync()
-                            Presence = false
-                        }
-                    };
-                });
-
-            var themeRepositoryMock = new Mock<IThemeRepository>();
-            themeRepositoryMock.Setup(x => x.Add(It.IsAny<Theme>()))
-                .Callback<Theme>(x =>
-                {
-                    x.Id = 5;
-                    x.Name = "ExampleName";
-                });
-
 
-            var mentorReposutoryMock = new Mock<IMentorRepository>();
-            var mentorReposutoryMockWrong = new Mock<IMentorRepository>();
-            mentorReposutoryMock.Setup(x => x.GetMentorByIdAsync(mentor.Id)).ReturnsAsync(mentor);
-            mentorReposutoryMockWrong.Setup(x => x.GetMentorByIdAsync(mentorWrong.Id)).ReturnsAsync(default(Mentor));
-
-            var studentGroupRepositoryMock = new Mock<IStudentGroupRepository>();
-            studentGroupRepositoryMock.Setup(x => x.GetByIdAsync(studentGroup.Id)).ReturnsAsync(studentGroup);
-            studentGroupRepositoryMock.Setup(x => x.GetGroupStudentsIds(studentGroup.Id)).ReturnsAsync(new List<long?> { 11, 14 });
+            _mentorRepositoryMock.Setup(x => x.GetMentorByIdAsync(mentor.Id)).ReturnsAsync(mentor);
 
             var studentGroupRepositoryMockWrong = new Mock<IStudentGroupRepository>();
-            studentGroupRepositoryMockWrong.Setup(x => x.GetByIdAsync(studentGroupWrong.Id)).ReturnsAsync(default(StudentGroup));
             studentGroupRepositoryMockWrong.Setup(x => x.GetGroupStudentsIds(createLessonDtoWrongLessonVisitsStudent.StudentGroupId)).ReturnsAsync(new List<long?> { 11 });
 
-            var visitRepositoryMock = new Mock<IVisitRepository>();
+            var lessonService = new LessonService(
+                unitOfWork: _unitOfWorkMock.Object,
+                mapper: _mapper,
+                currentUserService: _currentUserServiceMock.Object);
 
-            _unitOfWorkMock.Setup(x => x.LessonRepository).Returns(lessonRepositoryMock.Object);
-            _unitOfWorkMock.Setup(x => x.ThemeRepository).Returns(themeRepositoryMock.Object);
-            _unitOfWorkMock.Setup(x => x.MentorRepository).Returns(mentorReposutoryMock.Object);
-            _unitOfWorkMock.Setup(x => x.StudentGroupRepository).Returns(studentGroupRepositoryMock.Object);
-            _unitOfWorkMock.Setup(x => x.VisitRepository).Returns(visitRepositoryMock.Object);
+            #endregion
+            //Act 
+            var resultWithWrongLessonVisit = await lessonService.CreateLessonAsync(createLessonDtoWrongLessonVisitsStudent);
 
-            _currentUserServiceMock = GetCurrentUserAsExistingStudent();
+            //Assert
+            resultWithWrongLessonVisit.Error.Code.Should().BeEquivalentTo(ErrorCode.NotFound);
+        }
+
+        [Fact]
+        public async Task CreateLessonAsync_EmptyLessonVisit_ShouldReturnValidationError()
+        {
+            //Arrange
+            #region DATA
+            Mentor mentor = new Mentor() { Id = 2 };
```

Comment:
In some cases you're using constants, in others you don't
![asdasddas](https://user-images.githubusercontent.com/38729881/110019900-a0d33d80-7d31-11eb-8d1a-24024edd6434.png)

Please replace the magic number

Reply:
done